model/connection.js

```diff
@@ -157,6 +157,20 @@ class Connection extends ParchKey {
 
         return valid;
     }
+
+    /**
+     * Draw this Connection on the PaperScope's project.
+     *
+     * @param {PaperScope}  paperScope  The PaperScope to contain this
+     *                                  connection.
+     */
+    print(paperScope) {
+        if (this.segments) {
+            this.segments.forEach(value => {
+                value.print(paperScope);
```

Comment:
nit: convert `value` to `segment` since its a better description of the individual unit

Reply:
It's been changed.